udata_front/theme/gouvfr/assets/js/api/reports.ts

```diff
@@ -0,0 +1,36 @@
+import { api, type User } from "@etalab/data.gouv.fr-components";
+import { AxiosResponse } from "axios";
+
+export type ReportReason = string;
+
+export type ReportReasons = Record<string, string>;
+
+export async function getReportReasons() {
+  const res = await api.get<ReportReasons>("/reports/reasons/");
+  return res.data;
+}
+
+export type ObjectType = 'Dataset';
```

Comment:
I put Dataset only because that was the current configuration. But it will updated to do https://github.com/datagouv/udata-front/pull/436#discussion_r1675605291